config/config.go

```diff
@@ -13,6 +13,10 @@ type Config struct {
 	ShutdownTimeout            time.Duration `envconfig:"SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	OTExporterOTLPEndpoint		string		  `encconfig:"OTEL_EXPORTER_OTLP_ENDPOINT"`
+	OTServiceName				string		  `encconfig:"OTEL_SERVICE_NAME"`
+	OTBatchTimeout				time.Duration `encconfig:"OTEL_BATCH_TIMEOUT"`
+
```

Comment:
There are a few files in this repo that are not formatted correctly. Please run `go fmt ./...` from the root of the repo.